### DISCUSSION TITLE
Reciever Rssi Correction

### DISCUSSION THREAD

**Discussion by rookiebeotch** [open-sdr/openwifi#422] [Q&A] [answered]
Can you explain what these rssi correction table numbers represent?

inline int rssi_correction_lookup_table(u32 freq_MHz)
{
	int rssi_correction;

	if (freq_MHz<2412) {
		rssi_correction = 153;
	} else if (freq_MHz<=2484) {
		rssi_correction = 153;
	} else if (freq_MHz<5160) {
		rssi_correction = 153;
	} else if (freq_MHz<=5240) {
		rssi_correction = 145;
	} else if (freq_MHz<=5320) {
		rssi_correction = 145;
	} else {
		rssi_correction = 145;
	}

	return rssi_correction;
}

**Comment by JiaoXianjun** [comment, marked as answer]
https://github.com/open-sdr/openwifi/issues/405

**Comment by rookiebeotch** [comment]
thank-you
